libpod/healthcheck_linux.go

```diff
@@ -186,3 +187,21 @@ func (c *Container) hcUnitName(isStartup, bare bool) string {
 	}
 	return unitName
 }
+
+func (c *Container) sendHealthCheckLogToJournal(status string, log string) error {
```

Comment:
We have an existing healthcheck event, right? Can you extend that with optional fields instead of creating a new event for the log?

Reply:
Yes, but then the target could be affected by `events_logger`. I don't think this is a problem, but I wonder why `HealthCheckResults` were originally stored in a separate file.